### ISSUE TITLE
Runtime error when filter is on storage_filename

### ISSUE THREAD

**Issue by szymonk211** [heartexlabs/label-studio#4941]
**Describe the bug**
I have a label-studio instance and dataset synced from Google Cloud Storage. When I set filter value "storage_filename contains [value]" I got an error:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1955, in add_fields
    join_info = self.setup_joins(name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1648, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1562, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/label-studio/label_studio/data_manager/api.py", line 227, in get
    queryset = self.get_task_queryset(request, prepare_params)
  File "/label-studio/label_studio/data_manager/api.py", line 195, in get_task_queryset
    return Task.prepared.only_filtered(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 661, in only_filtered
    return queryset.prepared(prepare_params=prepare_params)
  File "/label-studio/label_studio/data_manager/managers.py", line 472, in prepared
    queryset = apply_filters(queryset, prepare_params.filters, project, request)
  File "/label-studio/label_studio/data_manager/managers.py", line 350, in apply_filters
    value_type = type(queryset.values_list(field_name, flat=True)[0]).__name__
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 867, in values_list
    clone = self._values(*_fields, **expressions)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 835, in _values
    clone.query.set_values(fields)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 2272, in set_values
    self.add_fields(field_names, True)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1982, in add_fields
    raise FieldError("Cannot resolve keyword %r into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id
``` 

**To Reproduce**
1. Go to main view of some labeling project
2. Click on "filters"
3. Click on "add filters"
4. In "where" select "Storage filename"
5. Write anything in third field after "contains" from second field

**Expected behavior**
Files filtered by selected conditions.

**Screenshots**
Files are not filtered and there is runtime error.

**Environment (please complete the following information):**
 - docker container on Alpine VM
 - Label Studio Community Version 1.9.1

**Additional context**
This is only sensible way I found to group files in some ways. For example I want to select all images from one folder and give them all the same label.

**Comment by hogepodge** [comment]
This is definitely a bug, and I've filed an internal report. We'll follow up when it's been fixed. Thanks for reporting it.

**Comment by hogepodge** [comment]
We have a fix in process. We'll update this when we have more information. Also reported in #4824.

**Comment by hogepodge** [comment]
Fixed upstream, this should be available in an upcoming Label Studio release (likely 1.10.3+, since this patch was added after the 1.10.2 cut)

**Comment by jombooth** [comment]
this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)
